ddtrace/contrib/elasticsearch/patch.py

```diff
@@ -45,14 +48,19 @@ def _es_modules():
             pass
 
 
+def get_version(elasticsearch):
+    return getattr(elasticsearch, "__version__", "0.0.0")
+
+
 # NB: We are patching the default elasticsearch.transport module
 def patch():
     for elasticsearch in _es_modules():
         _patch(elasticsearch)
 
 
 def _patch(elasticsearch):
-    if getattr(elasticsearch, "_datadog_patch", False):
+    # elasticsearch 8 is not yet supported
+    if getattr(elasticsearch, "_datadog_patch", False) or get_version(elasticsearch) >= (8, 0, 0):
```

Comment:
`get_version()` can return a string. It should always return a tuple.

Reply:
got it, I'll make sure it can handle either a tuple or str